config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -31,7 +31,8 @@ presubmits:
         - --node-test-args=--kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"
         - --node-tests=true
         - --provider=gce
-        - --test_args=--nodes=8 --focus="\[NodeConformance\]" --skip="\[Flaky\]|\[Slow\]|\[Serial\]"
+        # Explicitly enable probing container test to make sure we run against dockershim (https://github.com/kubernetes/kubernetes/issues/96463)
+        - --test_args=--nodes=8 --focus="\[NodeConformance\]|\[k8s\.io\] Probing container should be restarted startup probe fails" --skip="\[Flaky\]|\[Slow\]|\[Serial\]"
```

Comment:
Is there a more generic filter we can use here instead of specifcally the startup probe error?

Reply:
We could test more things, but just went on recommendation from @derekwaynecarr. Might be worth adding a new `[dockershim]` test label which would allow us to be more generic here.. thoughts?